src/main/g8/$name$.service

```diff
@@ -1,14 +0,0 @@
-[Unit]
-Description=$name;format="cap"$
-
-[Service]
-User=$stack_name_prefix$
-Group=$stack_name_prefix$
-Restart=no
```

Comment:
Looking at what [I think is the template code](https://github.com/sbt/sbt-native-packager/blob/master/src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template#L11), in my opinion it needs to be overridden to not restart as you expect the `ec2` instance to die to notice an issue if the jvm crash or could not start.